app/main.py

```diff
@@ -1 +1,58 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> any:
+        if self.is_hungry is True:
+            self.is_hungry = not self.is_hungry
+            print(f"Eating {self.appetite} food points...")
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+    def __init__(
+            self,
+            name: str,
+            appetite: int = 3,
+            is_hungry: bool = True
+    ) -> None:
+        super().__init__(name, is_hungry)
+        self.appetite = appetite
```

Comment:
Pass `appetite` to the `super().__init__` and do not redefine it out of this statement.

Reply:
when i pass apeetite  in super, i get uninspected argumen. please, explain how it work?

Reply:
how i must pass apeetite in super?